src/GoogleSignin.android.js

```diff
@@ -33,40 +33,37 @@ class GoogleSigninButton extends Component {
   render() {
     const { style, ...props } = this.props;
 
-    return (
-      <RNGoogleSigninButton style={[{ backgroundColor: 'transparent' }, style]} {...props} />
-    );
+    return <RNGoogleSigninButton style={[{ backgroundColor: 'transparent' }, style]} {...props} />;
   }
 }
 
 GoogleSigninButton.Size = {
   Icon: RNGoogleSignin.BUTTON_SIZE_ICON,
   Standard: RNGoogleSignin.BUTTON_SIZE_STANDARD,
-  Wide: RNGoogleSignin.BUTTON_SIZE_WIDE
+  Wide: RNGoogleSignin.BUTTON_SIZE_WIDE,
 };
 
 GoogleSigninButton.Color = {
   Auto: RNGoogleSignin.BUTTON_COLOR_AUTO,
   Light: RNGoogleSignin.BUTTON_COLOR_LIGHT,
-  Dark: RNGoogleSignin.BUTTON_COLOR_DARK
+  Dark: RNGoogleSignin.BUTTON_COLOR_DARK,
 };
 
+
 class GoogleSigninError extends Error {
   constructor(error, code) {
     super(error);
     this.name = 'GoogleSigninError';
-    this.code  = code;
+    this.code = code;
   }
 }
 
 class GoogleSignin {
-
-  constructor() {
-    this._user = null;
-  }
+  _user = null;
+  signinIsInProcess = false;
 
   hasPlayServices(params = { autoResolve: true }) {
-    return RNGoogleSignin.playServicesAvailable(params.autoResolve);
+    return Promise.resolve(true);
```

Comment:
What if Play Services are not installed in the device?

Maybe this method should be deprecated all together since there might not be any reason to use Google Sign In in the first place if Play Services are missing altogether.